tests/cpp_unit_tests/test_statistics.cpp

```diff
@@ -0,0 +1,820 @@
+                    PolarComplexRDV<symmetric_t> const polar{
+                        .magnitude = {.value = magnitude, .variance = magnitude_variance},
+                        .angle = {.value = 0.0, .variance = angle_variance}};
+
+                    CHECK(real(polar.value()) == doctest::Approx(polar.magnitude.value));
+                    CHECK(imag(polar.value()) == doctest::Approx(0.0));
+                }
+                SUBCASE("Perpendicular phase shift") {
+                    PolarComplexRDV<symmetric_t> const polar{
+                        .magnitude = {.value = magnitude, .variance = magnitude_variance},
+                        .angle = {.value = pi / 2, .variance = angle_variance}};
+
+                    CHECK(real(polar.value()) == doctest::Approx(0.0));
+                    CHECK(imag(polar.value()) == doctest::Approx(polar.magnitude.value));
+                }
+                SUBCASE("45deg phase shift") {
+                    PolarComplexRDV<symmetric_t> const polar{
+                        .magnitude = {.value = magnitude, .variance = magnitude_variance},
+                        .angle = {.value = pi / 4, .variance = angle_variance}};
+
+                    CHECK(real(polar.value()) == doctest::Approx(polar.magnitude.value * inv_sqrt2));
+                    CHECK(imag(polar.value()) == doctest::Approx(real(polar.value())));
+                }
+            }
+        }
+
+        SUBCASE("Conversion to DecomposedComplexRDV<symmetric_t>") {
+            for (auto const& [magnitude, magnitude_variance, angle_variance] :
+                 std::array{std::tuple{1.0, 1.0, 0.2}, std::tuple{2.0, 1.0, 0.2}, std::tuple{1.0, 3.0, 0.2},
+                            std::tuple{1.0, 2.0, 0.4}}) {
+                CAPTURE(magnitude);
+                CAPTURE(magnitude_variance);
+                CAPTURE(angle_variance);
+
+                SUBCASE("No phase shift") {
+                    PolarComplexRDV<symmetric_t> const polar{.magnitude = {.value = magnitude, .variance = magnitude},
+                                                             .angle = {.value = 0.0, .variance = angle_variance}};
+
+                    auto const decomposed = static_cast<DecomposedComplexRDV<symmetric_t>>(polar);
+
+                    CHECK(decomposed.real_component.value == doctest::Approx(polar.magnitude.value));
+                    CHECK(decomposed.imag_component.value == doctest::Approx(0.0));
+                    CHECK(decomposed.real_component.variance == doctest::Approx(polar.magnitude.variance));
+                    CHECK(decomposed.imag_component.variance ==
+                          doctest::Approx(magnitude * magnitude * polar.angle.variance));
+                    CHECK(real(decomposed.value()) == doctest::Approx(real(polar.value())));
+                    CHECK(imag(decomposed.value()) == doctest::Approx(imag(polar.value())));
+                }
+
+                SUBCASE("Perpendicular phase shift") {
```

Comment:
`Perpendicular` would be matching a rotation.
```suggestion
                SUBCASE("90deg phase shift") {
```